scripts/run-task

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -eu
+
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+
+Starts servers using docker-compose
```

Comment:
This might best to leave off and just `./scripts/run` with some instructions how to start from there in a README